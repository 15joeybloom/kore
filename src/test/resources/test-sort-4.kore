[]

module TEST4

  sort Map{S,S} []

endmodule []
