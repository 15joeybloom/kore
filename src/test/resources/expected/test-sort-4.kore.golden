Definition
    { definitionAttributes = Attributes []
    , definitionModules =
        Module
            { moduleName = (ModuleName "TEST4")
            , moduleSentences =
                [ (SentenceSortSentence SentenceSort
                    { sentenceSortName = (Id Object "Map")
                    , sentenceSortParameters =
                        [ (SortVariable (Id Object "S"))
                        , (SortVariable (Id Object "S"))
                        ]
                    , sentenceSortAttributes = Attributes []
                    })
                ]
            , moduleAttributes = Attributes []
            }
    }